### PR TITLE
[FIX] Fix compute_amount super and installation on readme.rst

### PR DESCRIPTION
**Pull request by hurrinico**
i made this pr in order to fix a little bug that i have found using globally swedish round and to set correctly the Readme.rst

### CODE REVIEW

File: account_invoice_rounding/models/account.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def _compute_swedish_rounding(self, invoice):
 
         if round_method[:7] != 'swedish':
             return {}
-
+        import pdb; pdb.set_trace()
```
Comment (leemannd) [inline account_invoice_rounding/models/account.py]: Can be removed ;)

**Comment by leemannd** [comment]
@grindtildeath Do you have the time to look at it?

**Comment by hurrinico** [comment]
@leemannd sorry for the pdb... fixed

**Comment by hurrinico** [comment]
I've change the decorator in order to align this module whit the [original one](https://github.com/odoo/odoo/blob/10.0/addons/account/models/account_invoice.py#L44)

**Review by eLBati — APPROVED** [review]

**Comment by eLBati** [comment]
@hurrinico Without this, invoice totals are not correctly computed, right?

**Comment by hurrinico** [comment]
yes @eLBati

**Pull request merged**